content/rs/administering/database-operations/slave-ha.md

```diff
@@ -5,44 +5,42 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-When you enable [database replication]({{< relref "/rs/concepts/high-availability/replication.md" >}})
-for your database, RS replicates your data to a slave node to make sure that your
-data is highly available. Whether the slave node fails or the master node fails
-and the slave is promoted to master, the remaining master node is a
-single point of failure.
+When you enable [database replication]({{< relref "/rs/concepts/high-availability/replication.md" >}}) for your database,
+RS replicates your data to a slave node to make sure that your data is highly available.
+If the slave node fails or if the master node fails and the slave is promoted to master,
+the remaining master node is a single point of failure.
 
-You can configure high availability for slave shards (slave HA) so that the cluster
-automatically migrates the slave shards to another available node. In practice, slave
-migration creates a new slave shard and replicates the data from the master shard to the
-new slave shard. For example:
+You can configure high availability for slave shards (slave HA) so that the cluster automatically migrates the slave shards to another available node.
+In practice, slave migration creates a new slave shard and replicates the data from the master shard to the new slave shard.
+For example:
 
 1. Node:2 has a master shard and node:3 has the corresponding the slave shard.
 1. Either:
 
     - Node:2 fails and the slave shard on node:3 is promoted to master.
-    - Node:3 fails and the master shard is no longer replicated.
+    - Node:3 fails and the master shard is no longer replicated to the slave shard on the failed node.
 
-1. If slave HA is enabled, a new slave shard is created on an available node
-    that does not hold the master shard.
+1. If slave HA is enabled, a new slave shard is created on an available node that does not hold the master shard.
 
     All of the constraints of shard migration apply, such as [rack-awareness]({{< relref "/rs/concepts/high-availability/rack-zone-awareness.md" >}}).
 
 1. The data from the master shard is replicated to the new slave shard.
 
 ## Configuring High Availability for Slave Shards
 
-You can enable slave HA using rladmin or using the REST API either for:
+You can enable slave HA using rladmin or using the REST API.
+Slave HA can be enabled or disabled at the cluster and database levels.
```

Comment:
"Slave HA **is** enabled or disabled at the cluster and database levels."

Reply:
Since you need both of them I think is better to say "it is" rather than "it can be"

Reply:
Please see the latest commit